packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1043,7 +1043,7 @@ oie.selfservice.unlock_user.failed.message = We are unable to unlock your accoun
 oie.selfservice.user.unlock.not.allowed = Self Service Unlock is not allowed at this time. Please contact support for assistance.
 
 ## Terminal errors
-idx.session.expired = The session has expired.
+idx.session.expired = You have been logged out due to inactivity. Refresh or return to the sign in screen.
```

Comment:
Can we have the IDX team's UX representative approve this messaging update in this PR please?

Reply:
We have confirmation from @jessicaallen-okta https://okta.slack.com/archives/CCA77QVL2/p1630440429159500?thread_ts=1630430199.144000&cid=CCA77QVL2